truffleruby/src/main/ruby/core/truffle/cext.rb

```diff
@@ -1000,32 +1000,6 @@ def rb_String(value)
     String(value)
   end
 
-  def RARRAY_PTR(array)
-    ArrayPointer.new(array)
-  end
-
-  class ArrayPointer
-
-    attr_reader :array
-
-    def initialize(array)
-      @array = array
-    end
-
-    def size
-      array.size
-    end
-
-    def [](index)
-      array[index]
-    end
-
-    def []=(index, value)
-      array[index] = value
-    end
-
-  end
```

Comment:
What happened to this? It's just not needed anymore because indexes are the same and Array methods have the same semantics as C ext reads/writes/size?